src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    _sub_job_download_inputs
+
+    # Make basecount batch string
+    # n.b. this *must* be sorted in the same order across all sub jobs as GATK assigns a sample
+    # index for the order in which the sample hdf5 files are provided, if they are not the same
+    # when calling PostProcessCNVCalls this will raise `The sample name is not the same for all of the shards`
+    batch_input=$(find in/basecounts/ -type f -name '*_basecount.hdf5' -exec basename {} \; \
+        | sort | sed 's/^/--input \/data\/basecounts\//g')
+
+    # Load the GATK docker image
+    docker load -i /home/dnanexus/in/GATK_docker/GATK*.tar.gz
+    export GATK_image=$(docker images --format="{{.Repository}} {{.ID}}" | grep "^broad" | cut -d' ' -f2)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)